corehq/apps/app_manager/util.py

```diff
@@ -752,3 +756,16 @@ def extract_instance_id_from_nodeset_ref(nodeset):
     if nodeset:
         matches = re.findall(r"instance\('(.*?)'\)", nodeset)
         return matches[0] if matches else None
+
+
+def does_app_have_mobile_ucr_v1_refs(app):
+    re_mobile_ucr_v1_all_refs = re.compile(MOBILE_UCR_V1_ALL_REFERENCES)
```

Comment:
Hey @zandre-eng , 
Yoliswa and I looked into the missing reference for one of the domains and found out that we need to check for manual references in the Case List/Detail section. This is available in the file `suite.xml` of the app. Below requested changes 
makes a check for that.

```suggestion
      # Check in Suite for references used in the case list/detail
      re_v1_case_list_references_pattern = re.compile(V1_CASE_LIST_REFERENCES_PATTERN)
      suite = app.create_suite()
      if re.search(re_v1_case_list_references_pattern, suite.decode()):
          return True                  

    re_mobile_ucr_v1_all_refs = re.compile(MOBILE_UCR_V1_ALL_REFERENCES)
```

Reply:
Good catch! I have added the necessary checks as suggested ([0cf6206](https://github.com/dimagi/commcare-hq/pull/35168/commits/0cf6206de2829c5b157acdfaf062aef9fd662fdf)).